package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git+https://github.com/apollostack/subscriptions-transport-ws.git"
   },
   "dependencies": {
-    "@types/node": "^6.0.38",
```

Comment:
well, it might be my mistake (that it's dependency and not devDependency.)
i don't think it should be removed but i suggest to move it to devDepenency and make sure package still importable..